2012/03/26 14:56:41 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000]: Syntax error or access violation: 1115 Unknown character set: 'utf8_general_ci'' in C:\xampp\yii-framework\db\CDbConnection.php:364
Stack trace:
#0 C:\xampp\yii-framework\db\CDbConnection.php(313): CDbConnection->open()
#1 C:\xampp\yii-framework\db\CDbConnection.php(291): CDbConnection->setActive(true)
#2 C:\xampp\yii-framework\base\CModule.php(372): CDbConnection->init()
#3 C:\xampp\yii-framework\base\CModule.php(86): CModule->getComponent('db')
#4 C:\xampp\yii-framework\gii\generators\model\ModelCode.php(54): CModule->__get('db')
#5 C:\xampp\yii-framework\web\CFormModel.php(40): ModelCode->init()
#6 C:\xampp\yii-framework\gii\CCodeGenerator.php(149): CFormModel->__construct()
#7 C:\xampp\yii-framework\gii\CCodeGenerator.php(61): CCodeGenerator->prepare()
#8 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#10 C:\xampp\yii-framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#11 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('')
#13 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('gii/model')
#14 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\working-sandbox\gitbox\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/working-sandbox/gitbox/index.php?r=gii/model
2012/03/26 14:57:46 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'gitbox-test.user' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `user`.
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (13)
2012/03/26 15:11:51 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\yii-framework\web\CController.php:328
Stack trace:
#0 C:\xampp\yii-framework\web\CController.php(301): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#3 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#7 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('view')
#9 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('user/view')
#10 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#11 C:\xampp\htdocs\working-sandbox\gitbox\index.php(13): CApplication->run()
#12 {main} REQUEST_URI=/working-sandbox/gitbox/index.php?r=user/view
2012/03/27 02:02:50 [error] [exception.CException] exception 'CException' with message 'UserIdentity and its behaviors do not have a method or closure named "getTableAlias".' in C:\xampp\yii-framework\base\CComponent.php:269
Stack trace:
#0 [internal function]: CComponent->__call('getTableAlias', Array)
#1 C:\xampp\yii-framework\db\ar\CActiveRecord.php(1457): UserIdentity->getTableAlias(true)
#2 C:\xampp\htdocs\working-sandbox\gitbox\protected\components\UserIdentity.php(25): CActiveRecord->findByAttributes(Array)
#3 C:\xampp\htdocs\working-sandbox\gitbox\protected\models\LoginForm.php(53): UserIdentity->authenticate()
#4 C:\xampp\yii-framework\validators\CInlineValidator.php(39): LoginForm->authenticate('password', Array)
#5 C:\xampp\yii-framework\validators\CValidator.php(192): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 C:\xampp\yii-framework\base\CModel.php(152): CValidator->validate(Object(LoginForm), NULL)
#7 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\SiteController.php(89): CModel->validate()
#8 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#9 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#10 C:\xampp\yii-framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#11 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('login')
#13 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('site/login')
#14 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\working-sandbox\gitbox\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/site/login
2012/03/27 02:05:48 [error] [exception.CDbException] exception 'CDbException' with message 'Table "user" does not have a column named "0".' in C:\xampp\yii-framework\db\schema\CDbCommandBuilder.php:568
Stack trace:
#0 C:\xampp\yii-framework\db\ar\CActiveRecord.php(1458): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 C:\xampp\htdocs\working-sandbox\gitbox\protected\components\UserIdentity.php(25): CActiveRecord->findByAttributes(Array)
#2 C:\xampp\htdocs\working-sandbox\gitbox\protected\models\LoginForm.php(53): UserIdentity->authenticate()
#3 C:\xampp\yii-framework\validators\CInlineValidator.php(39): LoginForm->authenticate('password', Array)
#4 C:\xampp\yii-framework\validators\CValidator.php(192): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\yii-framework\base\CModel.php(152): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\SiteController.php(89): CModel->validate()
#7 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#8 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#9 C:\xampp\yii-framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#10 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('login')
#12 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('site/login')
#13 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#14 C:\xampp\htdocs\working-sandbox\gitbox\index.php(13): CApplication->run()
#15 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/site/login
2012/03/27 02:29:38 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\yii-framework\web\CController.php:328
Stack trace:
#0 C:\xampp\yii-framework\web\CController.php(301): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#3 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#7 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('view')
#9 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('user/view')
#10 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#11 C:\xampp\htdocs\working-sandbox\gitbox\index.php(13): CApplication->run()
#12 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/user/view
2012/03/27 02:29:50 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\yii-framework\web\auth\CAccessControlFilter.php:158
Stack trace:
#0 C:\xampp\yii-framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\yii-framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#6 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('admin')
#8 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('user/admin')
#9 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#10 C:\xampp\htdocs\working-sandbox\gitbox\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/user/admin
2012/03/27 15:10:41 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.email" is not defined.' in C:\xampp\yii-framework\base\CComponent.php:131
Stack trace:
#0 C:\xampp\yii-framework\web\helpers\CHtml.php(2063): CComponent->__get('email')
#1 C:\xampp\yii-framework\web\helpers\CHtml.php(1791): CHtml::resolveValue(Object(LoginForm), 'email')
#2 C:\xampp\yii-framework\web\helpers\CHtml.php(1199): CHtml::activeInputField('text', Object(LoginForm), 'email', Array)
#3 C:\xampp\yii-framework\web\widgets\CActiveForm.php(568): CHtml::activeTextField(Object(LoginForm), 'email', Array)
#4 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\site\login.php(25): CActiveForm->textField(Object(LoginForm), 'email')
#5 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#6 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\yii-framework\web\CController.php(779): CController->renderPartial('login', Array, true)
#9 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\SiteController.php(93): CController->render('login', Array)
#10 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#11 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#12 C:\xampp\yii-framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#13 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('login')
#15 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('site/login')
#16 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#17 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#18 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/site/login
2012/03/27 15:16:58 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.email" is not defined.' in C:\xampp\yii-framework\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\working-sandbox\gitbox\protected\components\UserIdentity.php(25): CComponent->__get('email')
#1 C:\xampp\htdocs\working-sandbox\gitbox\protected\models\LoginForm.php(53): UserIdentity->authenticate()
#2 C:\xampp\yii-framework\validators\CInlineValidator.php(39): LoginForm->authenticate('password', Array)
#3 C:\xampp\yii-framework\validators\CValidator.php(192): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\yii-framework\base\CModel.php(152): CValidator->validate(Object(LoginForm), NULL)
#5 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\SiteController.php(89): CModel->validate()
#6 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#7 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#8 C:\xampp\yii-framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#9 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('login')
#11 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('site/login')
#12 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#13 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#14 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/site/login
2012/03/27 17:04:35 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'danish_satkut@hotmail.com' for key 'email'. The SQL statement executed was: INSERT INTO `user` (`group_id`, `username`, `password`, `email`, `email_setting`, `notify_with_post`, `auto_notify`, `show_smilies`, `show_img`, `show_img_sig`, `show_avatars`, `show_sig`, `access_keys`, `timezone`, `dst`, `time_format`, `date_format`, `language`, `style`, `num_posts`, `registered`, `registration_ip`, `last_visit`, `avatar`, `avatar_width`, `avatar_height`, `firstname`, `middlename`, `lastname`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20, :yp21, :yp22, :yp23, :yp24, :yp25, :yp26, :yp27, :yp28).
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\SiteController.php (136)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/03/27 17:04:35 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'danish_satkut@hotmail.com' for key 'email'. The SQL statement executed was: INSERT INTO `user` (`group_id`, `username`, `password`, `email`, `email_setting`, `notify_with_post`, `auto_notify`, `show_smilies`, `show_img`, `show_img_sig`, `show_avatars`, `show_sig`, `access_keys`, `timezone`, `dst`, `time_format`, `date_format`, `language`, `style`, `num_posts`, `registered`, `registration_ip`, `last_visit`, `avatar`, `avatar_width`, `avatar_height`, `firstname`, `middlename`, `lastname`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20, :yp21, :yp22, :yp23, :yp24, :yp25, :yp26, :yp27, :yp28)' in C:\xampp\yii-framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\yii-framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\yii-framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\SiteController.php(136): CActiveRecord->save()
#3 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): SiteController->actionRegister()
#4 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\yii-framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#6 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('register')
#8 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('site/register')
#9 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#10 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#11 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/site/register
2012/03/27 17:07:15 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'danish_satkut@hotmail.com' for key 'email'. The SQL statement executed was: INSERT INTO `user` (`group_id`, `username`, `password`, `email`, `email_setting`, `notify_with_post`, `auto_notify`, `show_smilies`, `show_img`, `show_img_sig`, `show_avatars`, `show_sig`, `access_keys`, `timezone`, `dst`, `time_format`, `date_format`, `language`, `style`, `num_posts`, `registered`, `registration_ip`, `last_visit`, `avatar`, `avatar_width`, `avatar_height`, `firstname`, `middlename`, `lastname`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20, :yp21, :yp22, :yp23, :yp24, :yp25, :yp26, :yp27, :yp28).
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\SiteController.php (136)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/03/27 17:07:15 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'danish_satkut@hotmail.com' for key 'email'. The SQL statement executed was: INSERT INTO `user` (`group_id`, `username`, `password`, `email`, `email_setting`, `notify_with_post`, `auto_notify`, `show_smilies`, `show_img`, `show_img_sig`, `show_avatars`, `show_sig`, `access_keys`, `timezone`, `dst`, `time_format`, `date_format`, `language`, `style`, `num_posts`, `registered`, `registration_ip`, `last_visit`, `avatar`, `avatar_width`, `avatar_height`, `firstname`, `middlename`, `lastname`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20, :yp21, :yp22, :yp23, :yp24, :yp25, :yp26, :yp27, :yp28)' in C:\xampp\yii-framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\yii-framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\yii-framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\SiteController.php(136): CActiveRecord->save()
#3 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): SiteController->actionRegister()
#4 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\yii-framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#6 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('register')
#8 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('site/register')
#9 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#10 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#11 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/site/register
2012/03/27 23:19:01 [error] [php] Missing argument 1 for User::validatePassword(), called in C:\xampp\htdocs\working-sandbox\gitbox\protected\components\UserIdentity.php on line 31 and defined (C:\xampp\htdocs\working-sandbox\gitbox\protected\models\User.php:130)
Stack trace:
#0 C:\xampp\yii-framework\validators\CInlineValidator.php(39): LoginForm->authenticate()
#1 C:\xampp\yii-framework\validators\CValidator.php(192): CInlineValidator->validateAttribute()
#2 C:\xampp\yii-framework\base\CModel.php(152): CInlineValidator->validate()
#3 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\SiteController.php(89): LoginForm->validate()
#4 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#5 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams()
#6 C:\xampp\yii-framework\web\CController.php(278): SiteController->runAction()
#7 C:\xampp\yii-framework\web\CController.php(257): SiteController->runActionWithFilters()
#8 C:\xampp\yii-framework\web\CWebApplication.php(277): SiteController->run()
#9 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController()
#10 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#11 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CWebApplication->run()
REQUEST_URI=/working-sandbox/gitbox/index.php/site/login
in C:\xampp\htdocs\working-sandbox\gitbox\protected\models\User.php (130)
in C:\xampp\htdocs\working-sandbox\gitbox\protected\components\UserIdentity.php (31)
in C:\xampp\htdocs\working-sandbox\gitbox\protected\models\LoginForm.php (55)
2012/03/28 02:04:58 [error] [php] trim() expects parameter 1 to be string, array given (C:\xampp\htdocs\working-sandbox\gitbox\protected\components\RequireLogin.php:22)
Stack trace:
#0 C:\xampp\yii-framework\base\CApplication.php(185): CWebApplication->raiseEvent()
#1 C:\xampp\yii-framework\base\CApplication.php(157): CWebApplication->onBeginRequest()
#2 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CWebApplication->run()
REQUEST_URI=/working-sandbox/gitbox/index.php/site/login
in C:\xampp\htdocs\working-sandbox\gitbox\protected\components\RequireLogin.php (22)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/04/09 00:44:49 [warning] [application] Failed to set unsafe attribute "folderDescription" of "Folder".
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php (71)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/04/09 00:44:50 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`gitbox-test`.`folder`, CONSTRAINT `folder_ibfk_1` FOREIGN KEY (`createdBy_fk`) REFERENCES `user` (`id`)). The SQL statement executed was: INSERT INTO `folder` (`folderName`) VALUES (:yp0).
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php (72)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/04/09 00:44:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`gitbox-test`.`folder`, CONSTRAINT `folder_ibfk_1` FOREIGN KEY (`createdBy_fk`) REFERENCES `user` (`id`)). The SQL statement executed was: INSERT INTO `folder` (`folderName`) VALUES (:yp0)' in C:\xampp\yii-framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\yii-framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\yii-framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(72): CActiveRecord->save()
#3 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#4 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#13 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#14 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#16 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
2012/04/09 00:45:20 [warning] [application] Failed to set unsafe attribute "folderDescription" of "Folder".
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php (71)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/04/09 00:45:43 [warning] [application] Failed to set unsafe attribute "folderDescription" of "Folder".
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php (71)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/04/09 00:47:19 [warning] [application] Failed to set unsafe attribute "folderDescription" of "Folder".
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php (71)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/04/09 00:48:15 [warning] [application] Failed to set unsafe attribute "folderDescription" of "Folder".
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php (71)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/04/09 00:48:18 [warning] [application] Failed to set unsafe attribute "folderDescription" of "Folder".
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php (71)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/04/09 00:53:10 [warning] [application] Failed to set unsafe attribute "folderDescription" of "Folder".
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php (71)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/04/09 00:54:31 [warning] [application] Failed to set unsafe attribute "folderDescription" of "Folder".
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php (71)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/04/09 00:54:47 [warning] [application] Failed to set unsafe attribute "folderDescription" of "Folder".
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php (71)
in C:\xampp\htdocs\working-sandbox\gitbox\index.php (22)
2012/04/09 01:32:09 [error] [exception.CException] exception 'CException' with message 'Property "FolderController.dateModified" is not defined.' in C:\xampp\yii-framework\base\CComponent.php:174
Stack trace:
#0 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(80): CComponent->__set('dateModified', 1333915329)
#1 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#2 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#3 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#4 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#5 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#6 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#7 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#8 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#9 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#11 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#12 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#13 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#14 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
2012/04/09 01:38:26 [error] [exception.Exception] exception 'Exception' with message 'DateTimeZone::__construct() [<a href='datetimezone.--construct'>datetimezone.--construct</a>]: Unknown or bad timezone (India/Mumbai)' in C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php:13
Stack trace:
#0 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php(13): DateTimeZone->__construct('India/Mumbai')
#1 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#2 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php(15): CController->renderPartial('_form', Array)
#5 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#6 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\yii-framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(87): CController->render('create', Array)
#10 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#11 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#12 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#13 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#14 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#18 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#20 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#21 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#22 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#23 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
2012/04/09 01:38:41 [error] [exception.Exception] exception 'Exception' with message 'DateTimeZone::__construct() [<a href='datetimezone.--construct'>datetimezone.--construct</a>]: Unknown or bad timezone (Indian/Mumbai)' in C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php:13
Stack trace:
#0 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php(13): DateTimeZone->__construct('Indian/Mumbai')
#1 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#2 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php(15): CController->renderPartial('_form', Array)
#5 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#6 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\yii-framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(87): CController->render('create', Array)
#10 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#11 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#12 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#13 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#14 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#18 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#20 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#21 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#22 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#23 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
2012/04/09 01:38:53 [error] [exception.Exception] exception 'Exception' with message 'DateTimeZone::__construct() [<a href='datetimezone.--construct'>datetimezone.--construct</a>]: Unknown or bad timezone (Indian/Chennai)' in C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php:13
Stack trace:
#0 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php(13): DateTimeZone->__construct('Indian/Chennai')
#1 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#2 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php(15): CController->renderPartial('_form', Array)
#5 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#6 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\yii-framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(87): CController->render('create', Array)
#10 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#11 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#12 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#13 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#14 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#18 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#20 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#21 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#22 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#23 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
2012/04/09 01:39:03 [error] [exception.Exception] exception 'Exception' with message 'DateTimeZone::__construct() [<a href='datetimezone.--construct'>datetimezone.--construct</a>]: Unknown or bad timezone (Indian/Calcutta)' in C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php:13
Stack trace:
#0 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php(13): DateTimeZone->__construct('Indian/Calcutta')
#1 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#2 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php(15): CController->renderPartial('_form', Array)
#5 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#6 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\yii-framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(87): CController->render('create', Array)
#10 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#11 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#12 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#13 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#14 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#18 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#20 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#21 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#22 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#23 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
2012/04/09 01:39:12 [error] [exception.Exception] exception 'Exception' with message 'DateTimeZone::__construct() [<a href='datetimezone.--construct'>datetimezone.--construct</a>]: Unknown or bad timezone (Indian/Kolkata)' in C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php:13
Stack trace:
#0 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php(13): DateTimeZone->__construct('Indian/Kolkata')
#1 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#2 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php(15): CController->renderPartial('_form', Array)
#5 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#6 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\yii-framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(87): CController->render('create', Array)
#10 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#11 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#12 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#13 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#14 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#18 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#20 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#21 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#22 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#23 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
2012/04/09 01:39:21 [error] [exception.Exception] exception 'Exception' with message 'DateTimeZone::__construct() [<a href='datetimezone.--construct'>datetimezone.--construct</a>]: Unknown or bad timezone (India/Kolkata)' in C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php:13
Stack trace:
#0 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php(13): DateTimeZone->__construct('India/Kolkata')
#1 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#2 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php(15): CController->renderPartial('_form', Array)
#5 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#6 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\yii-framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(87): CController->render('create', Array)
#10 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#11 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#12 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#13 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#14 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#18 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#20 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#21 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#22 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#23 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
2012/04/09 01:39:30 [error] [exception.Exception] exception 'Exception' with message 'DateTimeZone::__construct() [<a href='datetimezone.--construct'>datetimezone.--construct</a>]: Unknown or bad timezone (India/Calcutta)' in C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php:13
Stack trace:
#0 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php(13): DateTimeZone->__construct('India/Calcutta')
#1 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#2 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php(15): CController->renderPartial('_form', Array)
#5 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#6 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\yii-framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(87): CController->render('create', Array)
#10 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#11 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#12 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#13 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#14 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#18 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#20 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#21 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#22 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#23 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
2012/04/09 01:39:37 [error] [exception.Exception] exception 'Exception' with message 'DateTimeZone::__construct() [<a href='datetimezone.--construct'>datetimezone.--construct</a>]: Unknown or bad timezone (India/Chennai)' in C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php:13
Stack trace:
#0 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php(13): DateTimeZone->__construct('India/Chennai')
#1 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#2 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php(15): CController->renderPartial('_form', Array)
#5 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#6 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\yii-framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(87): CController->render('create', Array)
#10 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#11 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#12 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#13 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#14 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#18 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#20 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#21 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#22 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#23 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
2012/04/09 01:48:03 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.storagePath" is not defined.' in C:\xampp\yii-framework\base\CComponent.php:131
Stack trace:
#0 C:\xampp\yii-framework\base\CModule.php(88): CComponent->__get('storagePath')
#1 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php(12): CModule->__get('storagePath')
#2 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#3 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php(15): CController->renderPartial('_form', Array)
#6 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#7 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\yii-framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#10 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(85): CController->render('create', Array)
#11 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#12 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#13 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#14 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#15 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#16 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#17 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#19 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#21 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#22 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#23 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#24 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
2012/04/09 01:48:16 [error] [exception.CException] exception 'CException' with message 'Object configuration must be an array containing a "class" element.' in C:\xampp\yii-framework\YiiBase.php:198
Stack trace:
#0 C:\xampp\yii-framework\base\CModule.php(371): YiiBase::createComponent(Array)
#1 C:\xampp\yii-framework\base\CModule.php(86): CModule->getComponent('storage')
#2 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php(12): CModule->__get('storage')
#3 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#4 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php(15): CController->renderPartial('_form', Array)
#7 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#8 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\yii-framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#11 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(85): CController->render('create', Array)
#12 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#13 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#14 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#15 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#16 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#17 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#18 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#19 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#20 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#21 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#22 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#23 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#24 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#25 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
2012/04/09 02:12:18 [error] [php] Object of class CFile could not be converted to string (C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php:12)
Stack trace:
#0 C:\xampp\yii-framework\web\CController.php(866): FolderController->renderFile()
#1 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php(15): FolderController->renderPartial()
#2 C:\xampp\yii-framework\web\CBaseController.php(119): require()
#3 C:\xampp\yii-framework\web\CBaseController.php(88): FolderController->renderInternal()
#4 C:\xampp\yii-framework\web\CController.php(866): FolderController->renderFile()
#5 C:\xampp\yii-framework\web\CController.php(779): FolderController->renderPartial()
#6 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(85): FolderController->render()
#7 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#8 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams()
#9 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): FolderController->runAction()
#10 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#11 C:\xampp\yii-framework\web\CController.php(1144): CAccessControlFilter->filter()
#12 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): FolderController->filterAccessControl()
#13 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#14 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#15 C:\xampp\yii-framework\web\CController.php(257): FolderController->runActionWithFilters()
#16 C:\xampp\yii-framework\web\CWebApplication.php(277): FolderController->run()
#17 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController()
#18 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#19 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CWebApplication->run()
REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
in C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php (12)
in C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php (15)
in C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php (85)
2012/04/09 02:14:29 [error] [exception.CException] exception 'CException' with message 'Property "CFile.set" is not defined.' in C:\xampp\yii-framework\base\CComponent.php:174
Stack trace:
#0 C:\xampp\yii-framework\YiiBase.php(225): CComponent->__set('set', Array)
#1 C:\xampp\yii-framework\base\CModule.php(371): YiiBase::createComponent(Array)
#2 C:\xampp\yii-framework\base\CModule.php(86): CModule->getComponent('storage')
#3 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\_form.php(12): CModule->__get('storage')
#4 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#5 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\working-sandbox\gitbox\protected\views\folder\create.php(15): CController->renderPartial('_form', Array)
#8 C:\xampp\yii-framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#9 C:\xampp\yii-framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\yii-framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\yii-framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#12 C:\xampp\htdocs\working-sandbox\gitbox\protected\controllers\FolderController.php(85): CController->render('create', Array)
#13 C:\xampp\yii-framework\web\actions\CInlineAction.php(50): FolderController->actionCreate()
#14 C:\xampp\yii-framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#15 C:\xampp\yii-framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#16 C:\xampp\yii-framework\web\filters\CFilter.php(41): CFilterChain->run()
#17 C:\xampp\yii-framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#18 C:\xampp\yii-framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#19 C:\xampp\yii-framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#20 C:\xampp\yii-framework\web\CController.php(283): CFilterChain->run()
#21 C:\xampp\yii-framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\yii-framework\web\CWebApplication.php(277): CController->run('create')
#23 C:\xampp\yii-framework\web\CWebApplication.php(136): CWebApplication->runController('folder/create')
#24 C:\xampp\yii-framework\base\CApplication.php(158): CWebApplication->processRequest()
#25 C:\xampp\htdocs\working-sandbox\gitbox\index.php(22): CApplication->run()
#26 {main} REQUEST_URI=/working-sandbox/gitbox/index.php/folder/create
